writer/csv_writer.go

```diff
@@ -0,0 +1,53 @@
+package writer
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+type Headers []string
+
+func (h *Headers) getHeaderLine() string {
```

Comment:
nit: why a pointer receiver??? we're not updating the headers, so we could have `(h Headers)`

Reply:
changed